### ISSUE TITLE
d_loss and accuracy

### ISSUE THREAD

**Issue by Dr-Zhou** [corenel/pytorch-adda#22]
![image](https://user-images.githubusercontent.com/26479520/61129714-983c4c80-a4e7-11e9-86d4-64e977e35985.png)
My d_loss is 0，and my accuracy is 1 .

**Comment by LingkangYe** [comment]
I also meet the same problem。How do you solve it ?

**Comment by LingkangYe** [comment]
I also meet the same problem。How do you solve it ?

**Comment by Dr-Zhou** [comment]
No。I don't solve it。

**Comment by LingkangYe** [comment]
![image](https://user-images.githubusercontent.com/46863573/69059817-3b324380-0a51-11ea-86d2-3d95cff5a020.png)
After training, the accuracy is reduced.

**Comment by Dr-Zhou** [comment]
I think you can find another version of the paper. Maybe it receives good results.

**Comment by LingkangYe** [comment]
@Dr-Zhou  Alright. Thank you.

**Comment by Kang-Meng** [comment]
@Dr-Zhou Do u have any recommand, bro?

**Comment by ky-Pa** [comment]
> ![image](https://user-images.githubusercontent.com/26479520/61129714-983c4c80-a4e7-11e9-86d4-64e977e35985.png)
> My d_loss is 0，and my accuracy is 1 .

I also encountered the same problem. Is it resolved?

**Comment by LingkangYe** [comment]
I haven't solved it yet.------------------&nbsp;原始邮件&nbsp;------------------
发件人:&nbsp;&quot;ky-Pa&quot;<notifications@github.com&gt;
发送时间:&nbsp;2020年9月15日(星期二) 下午2:46
收件人:&nbsp;&quot;corenel/pytorch-adda&quot;<pytorch-adda@noreply.github.com&gt;;
抄送:&nbsp;&quot;LingkangYe&quot;<yelingkang@foxmail.com&gt;;&quot;Comment&quot;<comment@noreply.github.com&gt;;
主题:&nbsp;Re: [corenel/pytorch-adda] d_loss and accuracy (#22)
